src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -99,15 +99,15 @@ def action_file_copy(file_info):
 
 # pylint: disable=unused-argument
 def storage_blob_download_batch(client, source, destination, source_container_name, pattern=None, dryrun=False,
-                                progress_callback=None):
+                                progress_callback=None, max_connections=2):
```

Comment:
Can we defer the decision of default value to the SDK?

Reply:
yes indeed